script/test/cri-integration.sh

```diff
@@ -29,12 +29,15 @@ cd "${ROOT}"
 # FOCUS focuses the test to run.
 FOCUS=${FOCUS:-""}
 # REPORT_DIR is the the directory to store test logs.
-REPORT_DIR=${REPORT_DIR:-"/tmp/test-integration"}
+if [ $IS_WINDOWS -eq 0 ]; then
+  REPORT_DIR=${REPORT_DIR:-"/tmp/test-integration"}
+else
+  REPORT_DIR=${REPORT_DIR:-"C:/Windows/Temp/test-integration"}
+fi
 # RUNTIME is the runtime handler to use in the test.
 RUNTIME=${RUNTIME:-""}
 
 CRI_ROOT="${CONTAINERD_ROOT}/io.containerd.grpc.v1.cri"
```

Comment:
With `CONTAINERD_ROOT` no longer being set in `windows-periodic.yml`, won't this path be incorrect?

Reply:
Proper defaults for Windows have been set in ```utils.sh```, here: https://github.com/containerd/containerd/pull/6347/files#diff-ffe22935af27a0fc8014946fa256d68b3e9042327448c7204279925fb4799bfeR72

Which is then sourced by this file.

I added ```CONTAINERD_ROOT``` to the workflow as a temporary workaround to get the volume tests to pass. It is no longer needed after this change.

Reply:
Thanks, I missed that `utils.sh` is sourced above this line.